core-web/libs/sdk/client/src/lib/client/content/builders/collection/collection.ts

```diff
@@ -94,9 +117,9 @@ export class CollectionBuilder<T = unknown> {
     }
 
     /**
-     * The retrieved content will have the rendered HTML
+     * Retrieves content with rendered HTML.
```

Comment:
Are we sure about this? cc: @zJaaal

Reply:
@zJaaal Do you have the link to the docs?

Reply:
Done!